src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -924,6 +929,37 @@ public Unit getStrategicBombingRaidTarget(final Territory territory, final Colle
     return selected.get();
   }
 
+  @SuppressWarnings("serial")
```

Comment:
You should have eclipse generate a serialVersionUID for you instead of supressing this warning

Reply:
Done